pkg/adapter/mtping/runner.go

```diff
@@ -122,9 +125,21 @@ func (a *cronJobsRunner) Stop() {
 
 func (a *cronJobsRunner) cronTick(ctx context.Context, event cloudevents.Event) func() {
 	return func() {
+
+		event := event.Clone()
+		event.SetID(uuid.New().String())
+		target := cecontext.TargetFrom(ctx).String()
+		source := event.Context.GetSource()
+		time.Sleep(time.Duration(rand.Intn(500)) * time.Millisecond) // provide a delay so not all ping fired instantaneously distribute load on resources.
+
+		if a.Logger.Desugar().Core().Enabled(zap.DebugLevel) {
```

Comment:
this test is not needed.

Reply:
Strictly speaking you're right. I will remove. I was just thinking this would avoid a little formatting and calling when debug was off. But maybe it's just overkill.

Reply:
removed